src/components/Paddle/PaddleTransacionModal.tsx

```diff
@@ -266,14 +275,25 @@ export const PaddleTransacionModal = ({
           <Divider size="sm" label="OR" my="sm" labelPosition="center" />
           <Button
             variant="outline"
-            onClick={getTransaction}
+            onClick={() => getTransaction(captchaState.token)}
```

Comment:
Technically, the token might not be available here, better check for state === success.